File: python/01_05_Calculadora_basica/calculadora_basica.py
'''
Programa: 01_05_calculadora_basica
Simula el comportamiento de una calculadora basica El usuario Introducira 2 numeros enteros,
Los cuales se procesaran para mostrar por pantalla los resultado de una suma, resta, multiplicación,
división.

__author__ = "Alvaro Juanez M"
__version__ = "1.0"
__email__ = "alvarojuanezgit01@gmail.com"
__status__ = "Aprendiendo python"
'''

num_1 = int(input("Introduce el primer numero "))
num_2 = int(input("Introduce el segundo numero "))

# Operaciones basicas
suma = num_1 + num_2
resta = num_1 - num_2
multi = num_1 * num_2
division = int(num_1 / num_2)

#Imprime los resultados
print(num_1 , " + ", num_2, " = ", suma)
print(num_1 , " - ", num_2, " = ", resta)
print(num_1 , " * ", num_2, " = ", multi)
print(num_1 , " / ", num_2, " = ", division)

File: java/01_04_Calculdora_Basica/CalculadoraBasica.java
/*
 * CalculadoraBasica:  El programa simula el funcionamiento de una calculador sencilla
 * que opera sumas, restas, mutlipicación, division.
 * @author: Alvaro Juanez M
 * @version: 
 *
 */
import java.util.Scanner;

public class CalculadoraBasica{
    public static void main(String[] args){
        //Entrada de datos
        Scanner scanner = new Scanner(System.in);

        //Guarda el valor de las variables
        int num1, num2, total;

        //Inicializa el programa
        System.out.println("Introduce el primer numero");
        num1 = scanner.nextInt();
        System.out.println("Introduce el segundo numero");
        num2 = scanner.nextInt();

        scanner.close();    //cierre de scanner

        //Imprime el resultado de las operaciones matematicas
        System.out.println(num1 + " + " + num2 + " = " + (total = num1 + num2));
        System.out.println(num1 + " - " + num2 + " = " + (total = num1 - num2));
        System.out.println(num1 + " * " + num2 + " = " + (total = num1 * num2));
        System.out.println(num1 + " / " + num2 + " = " + (total = num1 / num2));
    }
}

